bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -442,7 +442,15 @@ private String getMergeReadyComment(String commitMessage, List<Review> reviews)
             }
             message.append("pushed to the `");
             message.append(pr.targetRef());
-            message.append("` branch. ");
+            message.append("` branch:\n\n");
+            divergingCommits.stream()
+                            .limit(10)
+                            .forEach(c -> message.append(" * ").append(c.hash().hex()).append(": ").append(c.message().get(0)).append("\n"));
+            if (divergingCommits.size() > 10) {
+                message.append(" * ... full list: ").append(pr.repository().webUrl(baseHash.hex(), pr.targetRef())).append("\n");
```

Comment:
Can we say something like `* ... and 5 more: `?

Reply:
Sure, but the link itself should show a full list, right?

Reply:
Yeah, I think that is the best